src/sagemaker/session.py

```diff
@@ -1225,6 +1225,115 @@ def logs_for_job(self, job_name, wait=False, poll=10):  # noqa: C901 - suppress
             billable_time = (description['TrainingEndTime'] - description['TrainingStartTime']) * instance_count
             print('Billable seconds:', int(billable_time.total_seconds()) + 1)
 
+    def logs_for_transform_job(self, job_name, wait=False, poll=10):  # noqa: C901 - suppress complexity warning
+        """Display the logs for a given transform job, optionally tailing them until the
+        job is complete. If the output is a tty or a Jupyter cell, it will be color-coded
+        based on which instance the log entry is from.
+        Args:
+            job_name (str): Name of the transform job to display the logs for.
+            wait (bool): Whether to keep looking for new log entries until the job completes (default: False).
+            poll (int): The interval in seconds between polling for new log entries and job completion (default: 5).
+        Raises:
+            ValueError: If waiting and the transform job fails.
```

Comment:
I'd remove "waiting and"

Reply:
I referred the annotation of `logs_for_job`. Since there isn't much difference in their logic, let me know if its annotation should be updated as well.

Reply:
good call - yeah, I'd update both spots